Source/bmalloc/libpas/src/libpas/pas_mte.h

```diff
@@ -552,16 +639,6 @@ PAS_IGNORE_WARNINGS_END
 
 #if PAS_OS(DARWIN)
 
-// We can't check whether PAS_ASSERT is defined since this header is included early
```

Comment:
This chunk was removed because it was only used by PAS_MTE_ZERO_FILL_PAGE, but now that we have pas_utils.h as an include we can just use PAS_ASSERT there instead.